story/cli.py

```diff
@@ -234,60 +234,69 @@ def user() -> dict:
 
 
 def print_command(command):
-    click.echo(click.style(f'$ {command}', fg='magenta'))
+    """Prints a command to the CLI."""
+
+    click.echo('  $ ' + click.style(command, fg='magenta'))
 
 
 def print_deprecated_warning(alternative):
-    click.echo(click.style('Warning: ', fg='yellow') +
-               'This command is deprecated and will be removed' +
-               ' in a future release. Please use ' +
-               click.style(f'$ {alternative}\n', fg='magenta'))
+    click.echo(
+        click.style('Warning: ', fg='yellow')
+        + 'This command is deprecated and will be removed'
+        + ' in a future release. Please use '
+        + click.style(f'$ {alternative}\n', fg='magenta')
+    )
 
 
 def assert_project(command, app, default_app, allow_option):
     if app is None:
-        click.echo(click.style('No Asyncy application found.', fg='red'))
+        click.echo(click.style('No Story Cloud application found.', fg='red'))
```

Comment:
```suggestion
        click.echo(click.style('No StoryScript Cloud application found.', fg='red'))
```

For now at least ...